apps/backend/api/src/app/app.module.ts

```diff
@@ -11,8 +11,15 @@ import { CharactersModule } from './characters/character.module';
 import { ForgeModule } from './forge/forge.module';
 import { ResourcesModule } from './resources/resources.module';
 import { GameResourcesModule } from './game-resources/game-resources.module';
+import { ResourcesUsedModule } from './resources-used/resources-used.module';
+import { ResourcesProducedModule } from './resources-produced/resources-produced.module';
+import { GameEventsModule } from './game-events/game-events.module';
+import { EventsModule } from './events/events.module';
+import { IsBonusMalusModule } from './is-bonus-malus/is-bonus-malus.module';
```

Comment:
Renommer pour enlever le `Is`/`is`

Reply:
Attention à bien adapter le frontend aussi vu que l'entité change de nom

Reply:
Fait